apps/ledger-live-desktop/src/newArch/features/Collectibles/hooks/useFetchOrdinals.ts

```diff
@@ -7,12 +7,13 @@ type Props = {
 
 const useFetchOrdinals = ({ account }: Props) => {
   const utxosAddresses = account.bitcoinResources?.utxos?.map(utxo => utxo.address).join(",") || "";
-  const { rareSats, inscriptions, isLoading, isError, isFetched } = fetchOrdinalsFromSimpleHash({
-    addresses: utxosAddresses,
-    threshold: 0,
-  });
+  const { rareSats, inscriptions, isLoading, isError, isFetched, error } =
+    fetchOrdinalsFromSimpleHash({
+      addresses: utxosAddresses,
+      threshold: 0,
+    });
```

Comment:
```suggestion
 return  fetchOrdinalsFromSimpleHash({
      addresses: utxosAddresses,
      threshold: 0,
    });
```

You can do that directly